vignettes/GLM_intro_course.Rmd

```diff
@@ -835,7 +835,7 @@ Note: you cannot use AIC in this case since the responses differ!
 
 ### You can predict outcomes for 2 different scales
 
-- for the scale of the **linear predictor**, e.g.:
+- for the scale of the **linear predictor** (default for glm), e.g.:
```

Comment:
```suggestion
- for the scale of the **linear predictor** (default for `glm()`), e.g.:
```

Reply:
👍 I notice that in most cases we're also specifying the NAMESPACE (i.e. `stats::glm()` and `spaMM::fitme()`), so I'll also do that here for consistency.